### ISSUE TITLE
Use config.defaultConfigJson to set inital values in etcd client/servers configs

### ISSUE THREAD

**Issue by stellarspot** [singnet/snet-daemon#74]
See the discussion: https://github.com/singnet/snet-daemon/pull/69#discussion_r226688798

The following shorter code does not read defaults from Viper:
```
	conf = &PaymentChannelStorageClientConf{}
	err = vip.UnmarshalKey(config.PaymentChannelStorageClientKey, conf)
```
There can be another way like using vip.Sub(key) to read default and non default values separately.

**Comment by vsbogd** [comment]
It is actually a problem similar to https://github.com/spf13/viper/issues/559
To workaround it and remove config duplication from code you can use config.SubWithDefault: https://github.com/singnet/snet-daemon/blob/551b58f6d3641c0a39e29d573fc213a650aef6e0/config/config.go#L204
method to get sub configuration into separate viper instance and then unmarshall it.

**Comment by stellarspot** [comment]
Yes, it is fixed in 6de4b53ece78b775dad4c2cde735c69267a031e3
